projects/archiver/src/tasks/notification/helpers/device-notifications-helpers.ts

```diff
@@ -31,12 +31,15 @@ const createScheduleNotificationEndpoint = (
 
 /**
  * TODO
- * it will work now only for Daily Editions in UK
+ * +3 hours from issue date is defaulted in.
+ * This will (obviously) work only for Daily Editions in UK
  * In the future we will need to make it more generic (for US and Australia)
  **/
-export const createScheduleTime = (issueDate: string): string => {
-    const THREE_AT_NIGHT = 'T03:00:00Z'
-    return `${issueDate}${THREE_AT_NIGHT}`
+export const createScheduleTime = (issueDate: string, offset = 3): string => {
+    const issueDateAsDate = moment.utc(issueDate, 'YYYY-MM-DD')
+    moment.locale('utc')
```

Comment:
What does `moment.locale('utc')` do? Does it affect the global state of something as a side-effect of this function? Is that what other code elsewhere in this program expects?

Reply:
That's an error.  Good spot.  Should be issueDateAsDate.locale('utc').  Retesting...